drizzlepac/haputils/product.py

```diff
@@ -663,6 +667,9 @@ def wcs_drizzle_product(self, meta_wcs):
                                   output=self.drizzle_filename,
                                   **drizzle_pars)
 
+        # Compute S_REGION keyword and add it to the header of this product
+        putils.compute_sregion(self.drizzle_filename)
+
         # Rename Astrodrizzle log file as a trailer file
         log.debug("Exposure image {}".format(self.drizzle_filename))
         try:
```

Comment:
Do the calls for s_region need to be in the SkycellExposure and SkycellProduct classes.?